app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import datetime
+import time
+
+
+def create_a_file_every_second():
+    while True:
+        now = time.strftime("%R:%S").split(":")
+        file_name = f"app-{now[0]}_{now[1]}_{now[2]}.log"
+
+        with open(file_name, "w") as f:
+            f.write(f"{datetime.datetime.now()}")
+
+        print(f"The file {file_name} created successfully")
```

Comment:
Due to statement, you need to print another message:
```
<timestamp> <filename>
```

Reply:
Example from readme:
```
# Output: 
# ...
# 2020-01-01 14:10:07.057405 app-14_10_7.log
# 2020-01-01 14:10:08.062182 app-14_10_8.log
# 2020-01-01 14:10:09.067101 app-14_10_9.log
# ...
```

Reply:
fixed